src/listeners/MainListener.js

```diff
@@ -24,7 +25,7 @@ module.exports = class MainListener extends EventListener {
       })
       this.log('[92mLavalink connection established!', 'Music')
     } else {
-      this.log('[91mLavalink connection didn\'t establish - LAVALINK_HOST not set.', 'Music')
+      this.log(`[91mLavalink connection didn\'t establish - Rrequired environment variable(s) (${lavalinkRequiredVariables.filter(variable => !process.env[variable]).join(', ')}) not set.`, 'Music')

```

Comment:
Possible improvement idea:
`Failed to establish Lavalink connection`

Reply:
outdated